src/components/Header/Header.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { TitleContainer } from './style';
+import Text from '../Text/Text';
+
+const Header = ({ title }) => (
```

Comment:
Do we expect this to be a header for only the sidebars? If so I might advocate for a less generic name, e.g., `SidebarHeader` just to disambiguate. What do you ya think?

Reply:
this is a great question for @twanlass hehehe, sorry to pass ya the monkey but do you imagine that these will be tightly scoped to the sidebar?

(monkey context: https://www.lce.com/pdfs/SPL_LP_Monkey_Management.pdf)

Reply:
Agree with @hamstu here! I'm thinking a more explicit name might work better in this case :).

Reply:
👋 Heya @hamstu, @gomezjuliana, and @ivanaszuber – awesome discussion here. My vote would be to scope this tighter / use a more specific name as well. _Something like_ `SidebarHeader` and `SidebarListItem` as shared feels clear.